content/blog/2023-11-03.md

```diff
@@ -0,0 +1,28 @@
++++
+title = "2023-11-03"
+date = 2023-11-03
++++
+
+---
+
+Today is Friday, day 307 of 365 (84.10%). We're in week 44 of 52 (84.61%). 
+
+## Planning
+
+What I am going to do today: 
+
+- [x] [continue DRF tutorial](https://github.com/encode/django-rest-framework/tree/master/docs/tutorial)
+
+## Activities executed
+
+Today I finished the parts 1 through 4 from the DRF tutorial.
+
+## Learnings
+
+Today I learned a lot about DRF, first I learned about serializers and how to use them to create and update snippets. I also saw how to use the ModelSerializer class to keep the code more concise.
+
+In part 2 of the tutorial I learned how to use more explicit status codes, the Request object instead of the HttpRequest object to handle methods other than POST and the Response object to render responses into the correct type. This helps with error handling and making the code more concise and readable.
+
+In the next part I rewrote the API views using class-based views and mixins and, alter on, generic class-based views, which shrank the code a lot.
```

Comment:
Here you could write to your code. Although our repo is not public, when you read it logged in on github, the navigation is straightforward.